docs/proposals/apiman-integration.md

```diff
@@ -0,0 +1,123 @@
+
+## Motivation
+Providers of services may have the need to control who and how their services are consumed.  This may be realized as policies defined by providers to control a service
+in any number of ways (e.g security, throttling/quota, billing and metrics)<sup>[1](#r1)</sup>. 
+
+## Constraints and Assumptions
+### Authentication
+The OAuth token identifying a user in OpenShift will be utilized by APIMan to associate service policy to their identity.  This will ensure a user's identity is 
+ubiquitous across the cluster when interacting with OpenShift and the APIMan management console.
+
+### Deployment
+#### Communication
+APIMan Components (e.g. gateway, management interface) integrated with the cluster will utilize mutual TLS for communication.
+
+#### On-Cluster Policy Storage
+[Origin Aggregated logging](https://github.com/openshift/origin-aggregated-logging)<sup>[4](#r4)</sup> will be deployed before APIMan in order to utilize the existing ElasticSearch cluster for policy storage.  
+The [ACL plugin](https://github.com/fabric8io/openshift-elasticsearch-plugin)<sup>[2](#r2)</sup> that is deployed
+as part of aggregated logging will be updated to allow access from the gateway.  ElasicSearch index management<sup>[3](#r3)</sup>  will be modified so
+that it will not cull APIMan policy data.
+
+#### Off-Cluster Policy Storage
+Cluster administrators can configure APIMan to use an alternate ElasticSearch cluster if desired.  Some organizations may have an existing ElasticSearch cluster that already defines
+policy, and adminstrators may desire to use it instead of the one provided with aggregated logging.  It is assumed in this configuration, the APIMan management interface will
+additionally rely on Kibana deployed off-cluster.  The multi-tenant features provided by aggregated logging for ElasticSearch and Kibana may not be available when configured to utilize these off-cluster
+dependencies.      
+
+## Use Cases
+* **UC01** As an API provider, I want to navigate from the OpenShift web console to the policy management interface, so I can define my service policies.
+* <s> **UC02** As an API provider, I want to associate an APIMan organization with an OpenShift project, so I can control policy between organizations.</s>
+* **UC03** As an API provider, I want to expose my service, so that it is consumable according to my management policy.  
+* **UC04** As a cluster administrator, I want a gateway to route API traffic only, so that it manages traffic based on a service's policy.
+* **UC05** As a cluster administrator, I want to deploy APIMan and its components reusing existing infrastructure components where possible (e.g. ElasticSearch), so I can minimize infrastructure components.
+* **UC06** As a cluster administrator, I want API management interface to be able to discover the route to Kibana, so it can be linked to from the management interface. 
+* **UC07** As an API consumer, I want to configure my services to consume other services
```

Comment:
I will add a place holder for this, since its a good question. @KurtStam are you able to comment and provide content on how a user will be allowed to use a service.